src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int lengthOne;
+    private int lengthTwo;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int lengthOne, int lengthTwo, int height) {
+        super(color);
+        this.lengthOne = lengthOne;
+        this.lengthTwo = lengthTwo;
+        this.height = height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + getArea() + " sq. units, "
```

Comment:
Please pay attention to your `draw()` method. In the output message, you are referring to the figure as a 'rectangle', but this class represents an 'isosceles trapezoid'. Please correct the figure name in the output message.